packages/coinstac-docker-manager/src/index.js

```diff
@@ -195,10 +195,10 @@ const startService = (serviceId, serviceUserId, opts) => {
  * @return {Object} Returns stream of docker pull output
  */
 const pullImage = (computation) => {
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve) => {
     docker.pull(computation, (err, stream) => {
       if (err) {
-        reject(err);
```

Comment:
So I don't particularly agree with this, I think the lower level function should reject if you have an image that is invalid, otherwise the behavior is inconsistent. _However_ the real solution is in comp registry to use `Promise.all` in a way to proceed even when one errors https://davidwalsh.name/promises-results.

You'll do that, then also pass an array of errors (if some reject) and results, and then deal w/ or display that.